### PR TITLE
adding pullapprove yaml file

### PR DESCRIPTION
**Pull request by rhiananthony**
### Purpose
The purpose of this PR is to add pullapprove functionality to this repo

---
### Changes
Addition of pullapprove v3 yaml file

---
### PR Checklist
_Please ensure the following when opening a PR:_

- [ ] This PR added or updated tests.
- [ ] This PR updated docstrings or documentation.
- [ ] This PR applied code style guidelines:
    - [Google style docstrings](https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html#example-google) Python code.
    - Mint WDL style guide for WDL.
- [ ] This PR considered generalizability beyond our own use case.

---
### Follow-up Discussions
_Please append follow-up discussions and issues during the review process below:_

- No follow-up discussions.

### CODE REVIEW

File: .pullapprove.yml [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+      required: 2
+      author_value: 1 # if author in group reduce number of approvals needed by this number
+      request_order: random  # reviewers will be chosen in a random order
+
+    reviewers:
+      teams:
+      - pipelines-software-engineers
+      - pipelines-devops
+
+  computational-review:
```
Comment (rexwangcc) [inline .pullapprove.yml]: I'm not sure if we want to enable CB reviews in this repo 🤔

File: .pullapprove.yml [hunk 2]
```diff
@@ -0,0 +1,42 @@
+version: 3
+
+groups:
+  pullapprove-admins:
+    conditions:
+    - "'.pullapprove.yml' in files"
+
+    reviews:
+      required: 1
```
Comment (rexwangcc) [inline .pullapprove.yml]: Does this mean we always need a 👍 from secops?

Comment (rhiananthony) [reply .pullapprove.yml]: Only for changes to the pullapprove file

File: .pullapprove.yml (hunk 1 above)
Comment (rhiananthony) [reply .pullapprove.yml]: Changed.

File: .pullapprove.yml (hunk 2 above)
Comment (rexwangcc) [reply .pullapprove.yml]: I should have realized that... `- "'.pullapprove.yml' in files"` that is the boundary of my knowledge. 👍

**Pull request merged**